src/ensembl/src/content/app/browser/drawer/drawer-views/DrawerTranscript.tsx

```diff
@@ -52,8 +52,8 @@ const DrawerTranscript: FunctionComponent<DrawerTranscriptProps> = (
             <Fragment>
               <p>
                 MANE Select transcripts match GRCh38 and are 100% identical
-                between Ensembl-GENCODE and RefSeq for 5' UTR, CDS, splicing and
-                3' UTR.
+                between Ensembl-GENCODE and RefSeq for 5&rsquo; UTR, CDS,
+                splicing and 3&rsquo; UTR.
```

Comment:
Aren’t actual Unicode symbols better than their representation in html entities?

Reply:
From a typography perspective it is good to be using HTML entities as some of the keyboard characters aren't the real characters that we need to be using. In this case the apostrophe is not the same as the right-side quote and they also look slightly different.

Reply:
I would always just key in the appropriate symbol. Shift-option-right_square_bracket gives you an `’`, which is the proper apostrophe/closing quotation mark.

But if we start considering typographically appropriate characters (slight shudder), then `5&rsquo; UTR` is not ever right. It's not 5-apostrophe (as in `it’s`, unicode U+0027) or 5-single_quote (unicode U+2019), but rather 5-prime (5′, unicode U+2032) 😞

Reply:
In that case we'll need to use the correct character then 😄 I think we will need to have a list of these commonly used special characters written somewhere for our reference.

Reply:
Or maybe refer this document: https://brajeshwar.github.io/entities/

Reply:
@azangru This has been disabled now